src/Telemetry/Telemetry.cs

```diff
@@ -426,5 +428,11 @@ public enum TelemetryErrorName
         Upsert,
         UpsertRollback,
     }
+
+    public class ServerProperties
```

Comment:
Can this be internal too?

Reply:
I started with internal, there was an error in SqlBindingingUtilities class that cannot access the object so made it public

Reply:
Was the error along the lines of "type has more visible modifier"? Because you'd need to make the method internal as well since it returns an internal type.